_episodes_rmd/04-bioconductor-vcfr.Rmd

```diff
@@ -21,17 +21,24 @@ source("../bin/chunk-options.R")
 knitr_fig_path("04-")
 ```
 
-<!-- example of rendered lesson https://datacarpentry.org/genomics-r-intro/02-r-basics/index.html -->
+## Packages in R -- what are they and why do we use them?
```

Comment:
I like this explanation! I noticed that we mention packages in the "R Basics continued" episode, but we don't explain it to this extent.  I wonder if this would be a better fit in an earlier episode than in the Bioconductor one.

Reply:
Sure, I think that probably makes sense. This is related to #170 -- if @JasonJWilliamsNY is ok with it I can pull this description out from here, and do a PR of the content on the R Basic continued lesson.

Reply:
Never mind -- might as well move fast and break things, eh? I pulled out the paragraph in commit [c28e053](https://github.com/datacarpentry/genomics-r-intro/pull/167/commits/c28e0530ed84cd66ebc83df8e96cb575ec674d93) and will PR it to the earlier lesson separately. @ytakemon let me know if there are any other adjustments you think I should make or if this is ready to merge.

Reply:
Looks good to me! Merging.